### PR TITLE
Add native-removed-history.md file

### PR DESCRIPTION
**Pull request by sgchoi5**
### Change Description ###

Add native-removed-history.md file for announcing removed functions and enums of Native API.

### CODE REVIEW

File: docs/application/native/tutorials/details/native-removed-history.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Removed functions and enumerations of Tizen Native API
+
+This page describes the functions and enumerations that were removed in Tizen 4.0 and 5.0.
```
Comment (mijin85cho) [inline docs/application/native/tutorials/details/native-removed-history.md]: It looks like this topic is going to list up all removed functions and enumerations not limited in from Tizen 4.0.
So, how about changing this sentence like `This topic describes the functions and enumerations that were removed`?

File: docs/application/native/tutorials/details/native-removed-history.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+| ------- | ------- | ------ | ---- | ---------- | ----------- |
+| Security - OpenSSL | SSLv2_method(), SSLv2_client_method(), SSLv2_server_method() | Mobile & Wearable | - | 3.0 | Security |
+| Base - Utils - i18n | i18n_timezone_in_daylight_time(), i18n_usearch_create(), i18n_ustring_to_title() | Mobile & Wearable | since 2.3.1 | 4.0 | No longer available | 
+| Web - WebView | ewk_context_preferred_languages_set(), ewk_settings_private_browsing_enabled_set(), ewk_settings_private_browsing_enabled_get() | Mobile | since 2.4 | 4.0 | No longer available |
+| Base - Glib | Gdbus APIs (g_bus_\*(), g_dbus_\*(), and g_test_dbus_\*())| Mobile & Wearable | since 2.3.2 | 5.0 | Security |
+| Content - Media Content | audio_meta_get_played_count(), audio_meta_get_played_time(), audio_meta_get_played_position(), audio_meta_set_played_count(), audio_meta_set_played_time(), audio_meta_set_played_position(), media_content_set_db_updated_cb(), video_meta_get_played_time(), video_meta_get_played_position(), video_meta_set_played_count(), video_meta_set_played_time(), video_meta_set_played_position()|Mobile & Wearable | since  2.4 | 5.0 | Better alternatives |
```
Comment (mijin85cho) [inline docs/application/native/tutorials/details/native-removed-history.md]: `Better alternatives` - How about writing to which alternative it is changed?

File: docs/application/native/tutorials/details/native-removed-history.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+# Removed functions and enumerations of Tizen Native API
+
+This page describes the functions and enumerations that were removed in Tizen 4.0 and 5.0.
+
+If you want to know the lifecycle of Tizen Native API, you need to read the contents of [API Versioning and Deprecation Policy of the Tizen Platform](https://developer.tizen.org/development/training/native-application/understanding-tizen-programming/api-versioning-and-deprecation-policy-tizen-platform) first.
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/tutorials/details/native-removed-history.md]: OK. I've fixed it. I will raise new Pull Request.

File: docs/application/native/tutorials/details/native-removed-history.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+| Multimedia - Recorder | RECORDER_POLICY_SOUND, RECORDER_POLICY_SOUND_BY_CALL, RECORDER_POLICY_SOUND_BY_ALARM of recorder_policy_e | Mobile & Wearable | since 3.0 | 5.0 | No longer available |
+| Multimedia - Camera | CAMERA_ERROR_SOUND_POLICY, CAMERA_ERROR_SOUND_POLICY_BY_CALL, CAMERA_ERROR_SOUND_POLICY_BY_ALARM of camera_error_e | Mobile & Wearable | since 3.0 | 5.0 | No longer available |
+| Multimedia - Camera | CAMERA_POLICY_SOUND, CAMERA_POLICY_SOUND_BY_CALL, CAMERA_POLICY_SOUND_BY_ALARM of camera_policy_e | Mobile & Wearable | since 3.0 | 5.0 | No longer available |
+| UI - EFL UTIL | efl_util_notification_window_level_error_cb(), efl_util_set_notification_window_level_error_cb(), efl_util_unset_notification_window_level_error_cb(), efl_util_window_screen_mode_error_cb(), efl_util_set_window_screen_mode_error_cb(), efl_util_unset_window_screen_mode_error_cb() | Mobile & Wearable | since 3.0 | 5.0 | No longer available |
+
+## Related information
+
+- [Tizen Native API Reference](https://developer.tizen.org/development/api-references/native-application)
```
[the remark this answers is not in the document]
Comment (sgchoi5) [reply docs/application/native/tutorials/details/native-removed-history.md]: OK. I've fixed it. I will raise new Pull Request.

File: docs/application/native/tutorials/details/native-removed-history.md (hunk 1 above)
Comment (sgchoi5) [reply docs/application/native/tutorials/details/native-removed-history.md]: OK. I've fixed it. I will raise new Pull Request.

File: docs/application/native/tutorials/details/native-removed-history.md (hunk 2 above)
Comment (sgchoi5) [reply docs/application/native/tutorials/details/native-removed-history.md]: It is diffcult to describe all alternatives in this document.
I will consider that....

**Comment by sgchoi5** [comment]
I will raise new Pull request with all fixes...

**Comment by sgchoi5** [comment]
I've updated it, but, I am not sure it is updated correctly. : ( There was a merge with master branch..

**Comment by mijin85cho** [comment]
In https://github.com/Samsung/tizen-docs/pull/315/files, I can see `docs/application/native/TOC.md` file and `docs/application/native/tutorials/details/details.md` files are newly updated, and native-removed-history.md's sentences are updated. Thanks for it : )

File: docs/application/native/tutorials/details/native-removed-history.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+| Multimedia - Recorder | RECORDER_POLICY_SOUND, RECORDER_POLICY_SOUND_BY_CALL, RECORDER_POLICY_SOUND_BY_ALARM of recorder_policy_e | Mobile & Wearable | since 3.0 | 5.0 | No longer available | - |
+| Multimedia - Camera | CAMERA_ERROR_SOUND_POLICY, CAMERA_ERROR_SOUND_POLICY_BY_CALL, CAMERA_ERROR_SOUND_POLICY_BY_ALARM of camera_error_e | Mobile & Wearable | since 3.0 | 5.0 | No longer available | - |
+| Multimedia - Camera | CAMERA_POLICY_SOUND, CAMERA_POLICY_SOUND_BY_CALL, CAMERA_POLICY_SOUND_BY_ALARM of camera_policy_e | Mobile & Wearable | since 3.0 | 5.0 | No longer available | - |
+| UI - EFL UTIL | efl_util_notification_window_level_error_cb(), efl_util_set_notification_window_level_error_cb(), efl_util_unset_notification_window_level_error_cb(), efl_util_window_screen_mode_error_cb(), efl_util_set_window_screen_mode_error_cb(), efl_util_unset_window_screen_mode_error_cb() | Mobile & Wearable | since 3.0 | 5.0 | No longer available | - |
+
+## Related information
+
+- [Tizen Native API Reference](../../api/)
```
Comment (mijin85cho) [inline docs/application/native/tutorials/details/native-removed-history.md]: During previous review, I mis-typed the path to go. Would you update the link to `(../../api/overview.md)`? Sorry for the inconvenience.

**Comment by sgchoi5** [comment]
Updated it

**Comment by sgchoi5** [comment]
This page was created to give developers notice (warning for removal).
It needs to be uploaded to public as soon as possible... actually, it was planned last year..

**Comment by mijin85cho** [comment]
Thanks for the explanation. As we talked on messenger, I'll update this PR on Site right after it's merge.

File: docs/application/native/tutorials/details/native-removed-history.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
+# Removed functions and enumerations of Tizen Native API
+
+This page describes the functions and enumerations that were removed since Tizen 3.0.
+
+If you want to know the lifecycle of Tizen Native API, you need to read the contents of [API Versioning and Deprecation Policy of the Tizen Platform](deprecation-policy.md) first.
+
+## Detailed information regarding removed functions and enumerations
```
Comment (annie-samsung) [inline docs/application/native/tutorials/details/native-removed-history.md]: I feel this heading is not required.
Instead add a text like this:
The following table provides detailed information about the removed functions and enumerations:

**Comment by sgchoi5** [comment]
Thanks for your comments. I've updated it.

**Comment by mijin85cho** [comment]
Before Annie re-check the updated file, 

1. `Change "Mobile & Wearable" to "Mobile, Wearable"` comment by Annie is not applied.
2. `The following table provides detailed information regarding removed functions and enumerations:` is still using head tag. Please remove `##` in front of this sentence. Also are you going to use "regarding" instead of suggested "about the"?

**Comment by sgchoi5** [comment]
Sorry, It was my mistake. I've updated it again.

**Comment by mijin85cho** [comment]
Thanks for the update : )

Hello, again, @annie-samsung .

When you agree with the updated file, please approve this PR.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**